tests/e2e/test_api.py

```diff
@@ -790,9 +786,8 @@ def test_query_filter() -> None:
         response_text = json_response["response"].lower()
         assert "openshift" in response_text
         assert "deploy" in response_text
-        response_words = response_text.split()
-        assert "foo" not in response_words
-        assert "bar" not in response_words
+        assert "foo" not in response_text
```

Comment:
the split was there on purpose, especially because "bar" is part of regular words

Reply:
I don't get it
```
>>> l = "bu ba bo"
>>> "be" not in l
True
>>> "be" not in l.split()
True
```
but ok.